webapp-kubernetes-go/go.mod

```diff
@@ -0,0 +1,8 @@
+module tmp
+
+go 1.17
```

Comment:
Go 1.17 is [EOL](https://endoflife.date/go). Probably want to use 1.18

Reply:
Fixed in 0a8a73a